guides.md

```diff
@@ -89,6 +89,10 @@ members. Skip the steps below at your discretion for low-stakes topics.
    [jitsi-android]: http://jitsi.org/android
    [jitsi-ios]: http://jitsi.org/ios
 
+📞 **Phone**
+  1. Use the private dial in number our [Jitsi documentation](https://github.com/hyphacoop/organizing-private/blob/master/documents/infrastructure/jitsi.md).
```

Comment:
```suggestion
  1. Use the private dial-in number from our [Jitsi documentation](https://github.com/hyphacoop/organizing-private/blob/master/documents/infrastructure/jitsi.md).
```

Reply:
```suggestion
  1. Dial our temporary semi-private conference line: `(647) 812-0386`
```

This is my preferred suggestion :)

I recall @dcwalk mentioning that there's some work to do to split/dedup content between the new technical wiki (not user-facing) and user-facing handbook docs. This feels related to that.

Reply:
related dedup of technical wiki/doc: https://github.com/hyphacoop/organizing-private/pull/9

Reply:
```suggestion
  1. Use the temporary, private dial-in number listed in our [technical Jitsi documentation](https://github.com/hyphacoop/organizing-private/blob/master/documents/infrastructure/jitsi.md).
```

ok, here's the simplest change, since given your lack of response, i maybe said something confusing. Reason: makes clear that someone is entered technical docs (so they don't assume they need to understand all the mumbo jumbo in there).

Reply:
I'm into this version! All looks good!

Reply:
@ASoTNetworks thanks man, but I think you merged the first suggestion instead of the final one that dc +1'd? Pls do read comments carefully, so we can all have full faith in one another :)